extensions/interactions/GraphInput/directives/graph-viz.component.ts

```diff
@@ -260,6 +292,25 @@ export class GraphVizComponent implements OnInit, AfterViewInit {
       this.vizContainer[0].getScreenCTM().inverse());
     this.state.mouseX = svgp.x;
     this.state.mouseY = svgp.y;
+    // The condition: currentMode === 2 signifies that
+    // the "add node" option in the graph is active.
+    // That is when we show the oppia-add-node-cursor
+    if (this.state.currentMode === 2) {
+      const dot = document.querySelector(
+        '.oppia-add-node-cursor') as HTMLDivElement;
+      const graphArea = this.element.nativeElement.querySelector(
```

Comment:
Avoid using the `querySelector` here, use `@ViewChild` instead. (Here and elsewhere)

Reply:
@Lawful2002 Done. PTAL. Thanks!